transactions/transactions.py

```diff
@@ -99,6 +99,35 @@ async def sign(self, ctx, user: discord.Member, team_name: str):
             except Exception as e:
                 await ctx.send(e)
 
+
+    @commands.guild_only()
+    @commands.command()
```

Comment:
if you update this line to `@commands.command(aliases=['re-sign'])`, then would `<p>re-sign` operate correctly as a command?

Reply:
I have added this, but it seems largely redundant to me, as `reZign` isn't jused anywhere in our nomenclature.